magicbot/magicrobot.py

```diff
@@ -386,6 +396,7 @@ def _create_components(self):
 
             # Automatically inject a logger object
             component.logger = logging.getLogger(m)
+            component.logger.addFilter(PeriodicFilter(self))
```

Comment:
Yes, because I said so. :p

I feel that having messages not show up when you expect them is a dangerous default behavior, and should only happen when you ask for it to happen.